src/panels/lovelace/cards/energy/hui-energy-devices-detail-graph-card.ts

```diff
@@ -64,7 +65,12 @@ export class HuiEnergyDevicesDetailGraphCard
 
   @state() private _compareEnd?: Date;
 
-  @state() private _hiddenStats = new Set<string>();
+  @storage({
+    key: "energy-devices-hidden-stats",
+    state: true,
+    subscribe: true,
```

Comment:
subscribe means that it will change in all browser tabs if it is changed in 1 tab, is that really what we want here?

Reply:
Right, does not really make sense here